src/common.h

```diff
@@ -2549,6 +2549,9 @@ win_set_focused_force(session_t *ps, win *w, switch_t val);
 void
 win_set_invert_color_force(session_t *ps, win *w, switch_t val);
 
+void
+win_set_the_blur_strength(session_t *ps, win *w, switch_t val);
```

Comment:
Shouldn't this just be called `win_set_blur_strength` like the other config parameters?
```suggestion
win_set_blur_strength(session_t *ps, win *w, switch_t val);
```
Would be a better fit in the `opts_set` category overall, as `blur-strength` is applied globally to all windows, not just a specific one.